commit 8c0ed9717810c1ace0bcf12b5235a9dc3bd04e82
Author: Yang Bo <yangbo@deepin.com>
Date:   Wed Aug 24 15:45:59 2016 +0800

    Add support for AMD Seattle I2C and APM X-Gene ACPI I2C device
    
    Signed-off-by: Yang Bo <yangbo@deepin.com>

diff --git a/drivers/i2c/busses/i2c-designware-platdrv.c b/drivers/i2c/busses/i2c-designware-platdrv.c
index cbd663d..c0ffb50 100644
--- a/drivers/i2c/busses/i2c-designware-platdrv.c
+++ b/drivers/i2c/busses/i2c-designware-platdrv.c
@@ -123,6 +123,8 @@ static const struct acpi_device_id dw_i2c_acpi_match[] = {
 	{ "80860F41", 0 },
 	{ "808622C1", 0 },
 	{ "AMD0010", ACCESS_INTR_MASK },
+    { "AMDI0510", 0 },
+    { "APMC0D0F", 0 },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, dw_i2c_acpi_match);
